x/cronos/keeper/precompiles/bank.go

```diff
@@ -134,11 +134,7 @@ func (bc *BankContract) Run(evm *vm.EVM, contract *vm.Contract, readonly bool) (
 		token := args[0].(common.Address)
 		addr := args[1].(common.Address)
 		// query from storage
-		balance := big.NewInt(0)
-		err = stateDB.ExecuteNativeAction(precompileAddr, nil, func(ctx sdk.Context) error {
-			balance = bc.bankKeeper.GetBalance(ctx, sdk.AccAddress(addr.Bytes()), EVMDenom(token)).Amount.BigInt()
-			return err
-		})
+		balance := bc.bankKeeper.GetBalance(stateDB.CacheContext(), sdk.AccAddress(addr.Bytes()), EVMDenom(token)).Amount.BigInt()
```

Comment:
Is this change supposed to be part of the IBC-go upgrade?

Reply:
need bump up [ethermint](https://github.com/crypto-org-chain/ethermint/compare/ae144329ecaf...4062d0cc2a32) and bank precompile query api also need